src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -330,6 +428,55 @@ class TestRequestRateManager : public TestLoadManagerBase,
  private:
   bool use_mock_infer_;
 
+  void TestCustomDataSetup(std::vector<ModelTensor>& tensors)
```

Comment:
nit: rename to `SetupCustomDataTest()` or `SetupTestCustomData()`?

Reply:
I agree. Starting with `Test` implies that this is a standalone test.

I've changed it to `CustomDataTestSetup`.